packages/jest-dev-server/src/index.js

```diff
@@ -91,22 +91,27 @@ async function outOfStin(block) {
   return result;
 }
 
-async function getIsPortTaken(config) {
-  let server;
-  const cleanupAndReturn = (result) =>
-    new Promise((resolve) => {
-      server.once("close", () => resolve(result)).close();
-    });
-  const val = await new Promise((resolve, reject) => {
-    server = net
-      .createServer()
-      .once("error", (err) =>
-        err.code === "EADDRINUSE" ? resolve(cleanupAndReturn(true)) : reject()
-      )
-      .once("listening", () => resolve(cleanupAndReturn(false)))
-      .listen(config.port, config.host);
```

Comment:
Sorry for the comment on this PR which has already been merged! But I think removing `config.host` from here, caused the issue: https://github.com/argos-ci/jest-puppeteer/issues/555. 

I've submitted a new PR addressing this 😁  https://github.com/argos-ci/jest-puppeteer/pull/556